app/components/documents/document-preview/version-card.hbs

```diff
@@ -1,62 +1,34 @@
-<li data-test-version-card-container class="auk-document-card {{if @isHighlighted "active"}}">
+<li ...attributes data-test-version-card-container class="auk-document-card auk-document-card--clickable {{if @isHighlighted "active"}}">
   <div class="auk-document-card__content">
-    <div
-      data-test-version-card-details
-      class="auk-document-card__details"
-    >
-      <Utils::AccessLevelColorBadge
-        @accessLevel={{this.accessLevel}}
-        class="auk-overline"
-      />
```

Comment:
:thinking: 
_Het weghalen van het toegangsniveau en de download knop op de versies tab is een design beslissing.
Persoonlijke mening hierover is dat de versie tab nu niet veel nut meer blijkt te bieden buiten vorige versie te moeten opendoen om de details ervan te kunnen bekijken of te kunnen bekijken/downloaden_

_In een typische flow zal er amper een visueel verschil te zien zijn tussen de kaartjes van versies van 1 document. Andere datums, en andere suffix in de naam._
:face_with_spiral_eyes: 

---


Dat gezegd zijnde, de transitie tussen bvb. BIS en TER door klikken op de kaartjes in de versies tab is niet meer zo proper. Het scherm gaat even naar de `loading` pagina, en daardoor krijg je precies een "witte flits" te zien omdat de hele route opnieuw geladen wordt. (de overgang van grijze achtergrond van de file naar witte loader naar grijze achtergrond valt op)
Mogelijk kan dit bij veelvuldig gebruik als storend worden ervaren?

Reply:
> Het weghalen van het toegangsniveau en de download knop op de versies tab is een design beslissing.
> Persoonlijke mening hierover is dat de versie tab nu niet veel nut meer blijkt te bieden buiten andere versies te kunnen opendoen om bepaalde details ervan te kunnen bekijken of te kunnen downloaden

@ValenberghsSven Waar heb je dit gelezen? De download knop zou moeten blijven voor **alle** documenten. Ook vorige versies. We hebben de download alleen verwijderd uit de widget. Niet uit de previewer.


Het belangrijkste is dat de gebruikers de verschillende versies kunnen raadplegen. We houden een oogje in het zeil bij de gebruikers mocht het toch als storend worden ervaren.